### PR TITLE
EG-1220, Fake welsh text

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/services/ScratchService.scala [hunk 1]
```diff
@@ -26,16 +26,20 @@ import repositories.ScratchRepository
...
+                                implicit val c: AppConfig) {
   val logger = Logger(getClass)
 
   def save(process: JsObject): Future[RequestOutcome[UUID]] =
     guidancePages(pageBuilder, securedProcessBuilder, process).fold(
       err => Future.successful(Left(err)),
-      _ =>
-        repository.save(process).map {
+      result =>
+        repository.save(result._3).map {
```
Comment (LesWilson) [inline app/services/ScratchService.scala:42]: Not a problem, just a comment.  I know using a tuple simplifies multiple returns, but it makes the code a bit less intuitive as you have to view the method called to see what entry 3 actually is.

Comment (cmrhmrc) [reply app/services/ScratchService.scala:42]: will update

File: app/services/package.scala [hunk 2]
```diff
@@ -14,60 +14,30 @@
...
-    case e: PhrasesParseError => ProcessError(s"Process Phrases section parse error, reason: ${e.msg}, index: ${e.id}", "")
-    case e: LinksParseError => ProcessError(s"Process Links section parse error, reason: ${e.msg}, index: ${e.id}", "")
-    case e: SharedDataInputStanza => ProcessError(s"Input/Question Stanza ${e.id} used on multiple pages ${e.pages}", e.id)
-    case e: PageRedirectNotSupported => ProcessError(s"Use of ChoiceStanza ${e.id} as a page redirect not supported", e.id)
-  }
-
-  implicit def processErrs(errs: List[GuidanceError]): List[ProcessError] = errs.map(toProcessErr)
-
-  def guidancePages(pageBuilder: PageBuilder, securedProcessBuilder: SecuredProcessBuilder, jsObject: JsObject): RequestOutcome[(Process, Seq[Page], JsObject)] =
+  def guidancePages(pageBuilder: PageBuilder, securedProcessBuilder: SecuredProcessBuilder, jsObject: JsObject)(implicit c: AppConfig): RequestOutcome[(Process, Seq[Page], JsObject)] =
```
Comment (LesWilson) [inline app/services/package.scala:27]: style warning in intellij - line exceeds 160 characters

Comment (cmrhmrc) [reply app/services/package.scala:27]: updated

File: test/models/ocelot/stanzas/InputStanzaSpec.scala [hunk 3]
```diff
@@ -66,264 +66,162 @@ class InputStanzaSpec extends BaseSpec {
...
-          val labels = LabelCache()
-
-          val (nxt, updatedLabels) = currencyInput.eval("33", labels)
-          updatedLabels.updatedLabels(expectedCurrencyStanza.label).english shouldBe Some("33")
-        case _ => fail
-      }
-
+      val input = Input(expectedCurrencyStanza, Phrase("",""), None, None)
+      val labels = LabelCache()
+      val (nxt, updatedLabels) = input.eval("33", labels)
```
Comment (LesWilson) [inline test/models/ocelot/stanzas/InputStanzaSpec.scala:71]: Intellij warning - nxt not used, replace with _

File: test/models/ocelot/stanzas/InputStanzaSpec.scala [hunk 4]
```diff
@@ -66,264 +66,162 @@ class InputStanzaSpec extends BaseSpec {
...
-          val labels = LabelCache()
-
-          val (nxt, updatedLabels) = currencyInput.eval("33", labels)
-          updatedLabels.updatedLabels(expectedCurrencyPoStanza.label).english shouldBe Some("33")
-        case _ => fail
-      }
-
+      val input = Input(expectedCurrencyPoStanza, Phrase("",""), None, None)
+      val labels = LabelCache()
+      val (nxt, updatedLabels) = input.eval("33", labels)
```
Comment (LesWilson) [inline test/models/ocelot/stanzas/InputStanzaSpec.scala:113]: as above for nxt

File: test/models/ocelot/stanzas/InputStanzaSpec.scala [hunk 5]
```diff
@@ -66,264 +66,162 @@ class InputStanzaSpec extends BaseSpec {
...
-        case input: DateInput =>
-          val labels = LabelCache()
-
-          val (nxt, updatedLabels) = input.eval("33", labels)
-          updatedLabels.updatedLabels(expectedDateStanza.label).english shouldBe Some("33")
-        case _ => fail
-      }
+      val input = Input(expectedDateStanza, Phrase("",""), None, None)
+      val labels = LabelCache()
+      val (nxt, updatedLabels) = input.eval("33", labels)
```
Comment (LesWilson) [inline test/models/ocelot/stanzas/InputStanzaSpec.scala:155]: as above

File: test/models/ocelot/stanzas/InputStanzaSpec.scala [hunk 6]
```diff
@@ -66,264 +66,162 @@ class InputStanzaSpec extends BaseSpec {
...
-        case input: TextInput =>
-          val labels = LabelCache()
-
-          val (nxt, updatedLabels) = input.eval("Hello", labels)
-          updatedLabels.updatedLabels(expectedTextStanza.label).english shouldBe Some("Hello")
-        case _ => fail
-      }
+      val input = Input(expectedTextStanza, Phrase("",""), None, None)
+      val labels = LabelCache()
+      val (nxt, updatedLabels) = input.eval("hello", labels)
```
Comment (LesWilson) [inline test/models/ocelot/stanzas/InputStanzaSpec.scala:182]: As above

File: test/models/ocelot/stanzas/InputStanzaSpec.scala [hunk 7]
```diff
@@ -66,264 +66,162 @@ class InputStanzaSpec extends BaseSpec {
...
-          val labels = LabelCache()
-
-          val (nxt, updatedLabels) = numberInput.eval("33", labels)
-          updatedLabels.updatedLabels(expectedNumberStanza.label).english shouldBe Some("33")
-        case _ => fail
-      }
-
+      val input = Input(expectedNumberStanza, Phrase("",""), None, None)
+      val labels = LabelCache()
+      val (nxt, newLabels) = input.eval("33", labels)
```
Comment (LesWilson) [inline test/models/ocelot/stanzas/InputStanzaSpec.scala:202]: as above

File: test/models/ocelot/stanzas/InputStanzaSpec.scala (hunk 3 above)
Comment (cmrhmrc) [reply test/models/ocelot/stanzas/InputStanzaSpec.scala:71]: done

File: test/models/ocelot/stanzas/InputStanzaSpec.scala (hunk 4 above)
Comment (cmrhmrc) [reply test/models/ocelot/stanzas/InputStanzaSpec.scala:113]: done

File: test/models/ocelot/stanzas/InputStanzaSpec.scala (hunk 5 above)
Comment (cmrhmrc) [reply test/models/ocelot/stanzas/InputStanzaSpec.scala:155]: done

File: test/models/ocelot/stanzas/InputStanzaSpec.scala (hunk 6 above)
Comment (cmrhmrc) [reply test/models/ocelot/stanzas/InputStanzaSpec.scala:182]: done

File: test/models/ocelot/stanzas/InputStanzaSpec.scala (hunk 7 above)
Comment (cmrhmrc) [reply test/models/ocelot/stanzas/InputStanzaSpec.scala:202]: done

**Review by LesWilson — CHANGES_REQUESTED** [review]
minor changes

**Review by LesWilson — APPROVED** [review]

**Pull request merged**